### PR TITLE
Tried to improve TSM connection test

### PR DESCRIPTION
**Pull request by jsmeix**
Tried to improve TSM connection test and
renamed misspelled 389_check_TSM_connexion.sh
into 389_check_TSM_connection.sh - cf.
https://github.com/rear/rear/issues/1534#issuecomment-351066398
and subsequent comments.

### CODE REVIEW

File: usr/share/rear/verify/TSM/default/389_check_TSM_connection.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# cf. https://github.com/rear/rear/issues/1642
+#
+# Regarding usage of '0<&6 1>&7 2>&8' see
+# "What to do with stdin, stdout, and stderr"
+# in https://github.com/rear/rear/wiki/Coding-Style
+#
+# First try "dsmc query session" because it is faster than "dsmc query mgmt":
+dsmc query session 0<&6 1>&7 2>&8 && return
+LogPrintError "Possibly no connection to TSM server ('dsmc query session' returns '$?')"
+# When "dsmc query session" failed try "dsmc query mgmt":
```
Comment (schabrolles) [inline usr/share/rear/verify/TSM/default/389_check_TSM_connection.sh]: We can remove `dsmc query mgmt` if we use `dsmc query session`.
The pupose here was to initiate a TSM connection to prompt for password if needed.
`dsmc query session` will do this job, and I agree with @dcz01, it is better than `dsmc query mgmt`

**Comment by jsmeix** [comment]
@schabrolles 
I removed 'dsmc query mgmt' and use only 'dsmc query session'
and improved the comments according to how I understand things.

**Comment by jsmeix** [comment]
@schabrolles 
in output/TSM/default/950_dsmc_save_result_files.sh
I noticed:
<pre>
    ... dsmc archive ...
...
ret=$?
# Error code 8 can be ignored
[ "$ret" -eq 0 -o "$ret" -eq 8 ]
StopIfError "Could not save result files with dsmc"
</pre>
so that I wonder if perhaps exit code 8
may have to be also ignored for 'dsmc query session'?

**Comment by schabrolles** [comment]
@jsmeix more information about TSM return code here http://publib.boulder.ibm.com/tividd/td/TSMC/GC32-0787-04/en_US/HTML/ans10000117.htm
I know somebody who will appreciate the "Warning" part. :-)

**Comment by jsmeix** [comment]
@schabrolles 
the current pull request state is that you still request changes
but I think I implemented what you had requested.
If you still need changes please tell me what you still need.

**Pull request merged**